.github/workflows/release.yml

```diff
@@ -0,0 +1,143 @@
+name: Release
+
+on:
+  push:
+    tags:
+    - "*-edge"
```

Comment:
Noting that this will miss non-edge (stable?) releases.

Reply:
hmm yeah, the helm chart publishing below also only deals with edge releases for now. But I think I can just use here a more general regex :+1: